demos/react/typescript/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@hpcc-js/demo-react-typescript",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@hpcc-js/wc-layout": "^0.1.0",
```

Comment:
This doesn't match the layout/package.json version `0.0.0`

Reply:
Resolved